data/simpy.gram

```diff
@@ -340,18 +346,31 @@ atom[expr_ty]:
     | a=STRING+ { asdl_seq_GET(a, 0) }
     | NUMBER
     | a='...' { _Py_Constant(Py_Ellipsis, NULL, EXTRA(a, token_type, a, token_type)) }
-list: '[' [star_full_expressions] ']'
-listcomp: '[' full_expression for_if_clauses ']'
-tuple: '(' [star_full_expression ',' [star_full_expressions]] ')'
-group: '(' (yield_expr | full_expression) ')'
-genexp: '(' expression for_if_clauses ')'
-set: '{' expressions '}'
-setcomp: '{' expression for_if_clauses '}'
-dict: '{' [kvpairs] '}'
-dictcomp: '{' kvpair for_if_clauses '}'
-kvpairs: kvpair (',' kvpair)* [',']
-kvpair: '**' bitwise_or | expression ':' expression
-for_if_clauses: ([ASYNC] 'for' star_targets 'in' expression ('if' expression)*)+
+
+list[expr_ty]:
+    | a='[' b=[star_full_expressions] c=']' { _Py_List(b, Load, EXTRA(a, token_type, c, token_type)) }
+listcomp[expr_ty]:
+    | a='[' b=full_expression c=for_if_clauses d=']' { _Py_ListComp(b, c, EXTRA(a, token_type, d, token_type)) }
+tuple[expr_ty]:
+    | a='(' b=[y=star_full_expression ',' z=[star_full_expressions] { seq_insert_in_front(p, y, z)} ] c=')' {
+        _Py_Tuple(b, Load, EXTRA(a, token_type, c, token_type)) }
+group[expr_ty]: '(' a=(yield_expr | full_expression) ')' { a }
```

Comment:
As this PR also changes the expressions, would it make sense to add some tests that exercise the `( yield )`  case? We can do them in a different PR

Reply:
I don't exactly understand, which case you mean? There is the `(yield a)` test. Is there something more than that that needs to be added?

Reply:
> There is the `(yield a)` test

My bad, I have missed it :S